frame/transaction-payment/src/lib.rs

```diff
@@ -445,6 +445,44 @@ where
 		}
 	}
 
+	/// Query information of a dispatch class, weight, and fee of a given encoded `Call`.
+	pub fn query_call_info(
+		encoded_call: Vec<u8>,
+	) -> Result<RuntimeDispatchInfo<BalanceOf<T>>, RuntimeString>
```

Comment:
```suggestion
	) -> RuntimeDispatchInfo<BalanceOf<T>>
```

Reply:
I see, since it's not dispatchable, it's ok to panic.